Casks/t/ttu-base-suite-2000.rb

```diff
@@ -0,0 +1,24 @@
+cask "ttu-base-suite-2000" do
+  version "20.00.04.00"
+  sha256 "663e8bd8f66f11fa35face2860a38de5a77f38b2954db93f5dc2db0b0cfb86ae"
+  
+  url "https://downloads.teradata.com/sites/default/files/2023-12/TeradataToolsAndUtilities-macosx-brew-#{version}.tar"
+  name "TTU BASE Suite #{version}"
+  desc "Teradata Tools and Utilities BASE Suite #{version}"
```

Comment:
The `desc` stanza is often cached, so it would be better to not include the version.

```suggestion
  desc "Teradata Tools and Utilities BASE Suite"
```